language_reference_manual.md

```diff
@@ -5,6 +5,76 @@
 ## Types
 
 ## Operators 
+### Arithmetic Operators
+The binary arithmetic operators are +, -, *, /, % .
+Integer division truncates any fractional part.
+The binary + and - operators have the same precedence, which is lower than the
+precedence of *, /, % . Arithmetic operators associate left to right.
+
+### Grammar
+
+```
+expr:
+expr PLUS expr  }
+| expr MINUS expr }
+| expr TIMES expr }
+| expr DIVIDE expr }
+| expr MODULO expr }
+```
+
+### Example Syntax
+
+```
+int x = 5 * (16 - 6);
+```
+
+### Assignment Operators
+The operator = assigns an expression to a variable. It assigns value on right of operator to the left.
+
+### Example Syntax
+
+```
+int x = 5;
+```
+
+### Comaparision Operators
+The relational operators are <, <=, >, >= . They all have the same precedence. 
+
+### Grammar
+
+```
+expr:
+expr EQ expr 
+| expr NEQ expr 
+| expr LT expr  
+| expr LEQ expr  
+| expr GT expr  
+| expr GEQ expr 
+```
+
+### Example Syntax
+
+```
+bool x = (1 == 1);
+```
+
+### Logical Operators
+The logical operators are and, or, not.
+
+### Grammar
+
+```
+expr:
+expr AND expr 
+| expr OR expr 
+| NOT expr  
+```
+
+### Example Syntax
+
+```
+if(x == y or x == z)
+```
 
```

Comment:
I think it might be good to have examples for every one of our operators in grammar. I know that might be overdoing it but I think it could be better to have too much detail than get dinged for not having enough.